app/Http/Controllers/ApplicationController.php

```diff
@@ -146,9 +145,16 @@ public function edit($id)
       $hear_about = Scholarship::getCurrentScholarship()->hear_about_options;
       $choices = Application::formatChoices($hear_about);
 
+      // We have to pass uploads to the view, so set null if there aren't any
+      if ($application['upload']) {
```

Comment:
Should this be wrapped in an `! empty()` check? It might also be nice to use `$application->upload` field access like we do elsewhere in this method.